src/main/main.ts

```diff
@@ -663,8 +663,8 @@ const createWindow = lazy(async () => {
         width: lastWindowSize?.width ?? 1280,
         height: lastWindowSize?.height ?? 720,
         backgroundColor: '#1A202C',
-        minWidth: 720,
-        minHeight: 640,
+        minWidth: 840,
```

Comment:
The min width was already quite large before. Is this really necessary?

E.g. my laptop has 1920x1080px screen with 125% DPI scaling, so an app window will be 768px (adjusted for DPI scaling) wide when snapped to the left/right side of the screen. With this change, chainner will be annoyingly larger than half.

Reply:
I suppose I could lower it back, but keep in mind things start overflowing off the screen when it becomes that small. If we ever add anything more to the header this could start becoming a big issue. I'm not really familiar with ways to resolve that issue. I guess we could hide things that don't matter when it becomes smaller?

Reply:
Yes, basically we need responsive design.